src/layout/layout.js

```diff
@@ -140,6 +139,12 @@
     return window.matchMedia(query);
   };
 
+  var mediaQuery = MaterialLayout.prototype.matchMedia_(
```

Comment:
It seems weird to be using instance methods for this. I would change `matchMedia_` to just be `MaterialLayout.matchMedia_`